tests/audio_test.py

```diff
@@ -35,3 +40,9 @@ def test_get_jingles(monkeypatch):
     begin, end = get_jingles(song_name)
     assert len(begin) == 20000
     assert len(end) == 10000
+
+def test_glue_tracks(monkeypatch):
+    first = MockAudioSegment.from_mp3("whatever")
+    second = MockAudioSegment.from_mp3("Anything else")
+    final = glue_tracks([(first, 0), (second, 1000)])
+    assert len(final) == 100000
```

Comment:
This test is not enough. It only tests the resulting audio is long enough, but not defines anything about the order, or the position.

Reply:
I created issue #6 for this